Tasks/AzureFunctionAppV1/task.loc.json

```diff
@@ -116,10 +116,20 @@
         "EditableOptions": "True"
       },
       "options": {
-        "DOCKER|microsoft/azure-functions-dotnet-core2.0:2.0": ".NET",
-        "DOCKER|microsoft/azure-functions-node8:2.0": "JavaScript"
+        "DOTNET|2.2": "DOTNET|2.2 (functionapp v2)",
+        "DOTNET|3.1": "DOTNET|3.1 (functionapp v3)",
+        "JAVA|8": "JAVA|8 (functionapp v2/v3)",
+        "JAVA|11": "JAVA|11  (functionapp v3)",
+        "NODE|8": "NODE|8 (functionapp v2)",
+        "NODE|10": "NODE|10 (functionapp v2/v3)",
+        "NODE|12": "NODE|12 (functionapp v3)",
+        "NODE|14": "NODE|14 (functionapp v3)",
+        "PYTHON|3.6": "PYTHON|3.6 (functionapp v2/v3)",
+        "PYTHON|3.7": "PYTHON|3.7 (functionapp v2/v3)",
+        "PYTHON|3.8": "PYTHON|3.8 (functionapp v3)"
```

Comment:
@Hazhzeng / @vrdmr  / @anirudhgarg  - Should this include Python 3.9?

Reply:
Since there is no response as well as the public doc does not refer the inclusion of python 3.9, I'. resolving this and proceeding. Please open a new issue if needed.